doc/ref_model/chapters/chapter08.rst

```diff
@@ -210,14 +209,133 @@ The API and Cloud Brokerage models are illustrated in the following diagrams:
 Requirements, Reference Architecture & Industry Standards Intersect
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The Communcations Service Provider is both a provider and consumer of Cloud based services.
+The Communcations Service Provider (CSP) is both a provider and consumer of Cloud based services.
 When the CSP is actings as:
 
 -  consumer, in which case the typical consideration is total cost of ownership as the consumption is to usually to support internal business operations: BSS/OSS systems;
 -  provider of cloud services, through operation of their own cloud or reselling of cloud services, in which case the typical consideration is margin (cost to offer services vs income received).
 
 These two stances will drive differing approaches to how a CSP would look to manage how it interacts within a Multi-Cloud environment.
 
+Single Pane of Glass
+~~~~~~~~~~~~~~~~~~~~
+
+As organisations spread their resources across on-premises, multiple clouds, and the Edge, the need for a single set of tools and processes to manage and operate across these Hybrid, Edge, and Multi-clouds (HEM clouds) is obvious as can be seen form the following simplistic sceanrio. 
+
+Scenario: An opertor has private clouds that it utilises for its workloads.
+
+- A: The operator has decided to utilise one or more public clouds for some of its workloads.
+- B: The operator has decided to utilise an edge clouds for some of its clients.
+- C: The operator has decided to create edge clouds for some of its clients.
+
+Scenario B can be treated being the same as Scenario A. Scenario C is akin to the private cloud except for location and control over the facilities at that location. For its worklooads, the operator will have to utilise the target clouds tools or APIs to create the necessary accounts, billing arrangements, quotas, etc. Then create the needed resources, such as VMs or kubernetes clusters, etc. Following up with creating needed storage, networking, etc. before onboarding the workload and operating it. This is complex even when the operator is delaing with say only one other cloud, in addition to operating its own cloud. The operator is faced with a number of challenges including acquiring a new set of skills, knowledge of APIs, tools, and the complexity of managing different policies, updates, etc. This becomes impossible to manage when incorporating more than one other cloud. Hence the need for a Single Pane of Glass
```

Comment:
@ulikleber I hope I am not saying that but rather pointing out that it would be impossible to manage if more than 2 clouds. Let us discuss in RM meeting. Thabks